orangecontrib/network/widgets/ownxsinglemode.py

```diff
@@ -150,13 +156,21 @@ def update_output(self):
 
     def _mode_masks(self):
         """Return indices of nodes in the two modes"""
+        self.Warning.ignoring_missing.clear()
```

Comment:
I bit of nitpicking.

Say that we have a variable with missing values and the warning indicator is shown. Besides this, say that `self.connect_value` then becomes equal to self.connector_value - 1` (I have no suitable data available and so haven't reproduced this, but I assume the scenario is possible). Next, the user changes the variable to one without missing values.

 `indicator_changed` calls `update_output`. `update_output` does not call `_mode_masks` (because of `if self.connect_value == self.connector_value - 1`, so the warning is not cleared).

Perhaps `self.Warning.ignoring_missing.clear()` should be called at the beginning of `update_output` instead of here.

Reply:
I think you're right, thanks for pointing that out - I've changed it